.github/workflows/deploy-standalone-plugins.yml

```diff
@@ -69,15 +86,20 @@ jobs:
       - name: Install npm dependencies
         run: npm ci
       - name: Building standalone plugins
+        if: ${{ matrix.directory != 'plugins' }}
         run: npm run build-plugins
       - name: Deploy Standalone Plugin - ${{ matrix.slug }}
         uses: 10up/action-wordpress-plugin-deploy@stable
         with:
           dry-run: ${{ matrix.dry-run }}
         env:
-          SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
-          SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+          # TODO Once the workflow is tested, we will remove the comment and use the secret SVN access.
+          #SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+          #SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+          # TODO Once the workflow is tested, we will remove this test credential.
+          SVN_PASSWORD: SVN_PASSWORD
+          SVN_USERNAME: SVN_USERNAME
           SLUG: ${{ matrix.slug }}
           VERSION: ${{ matrix.version }}
-          BUILD_DIR: ./build/${{ matrix.slug }}
-          ASSETS_DIR: ./build/${{ matrix.slug }}/.wordpress-org
+          BUILD_DIR: ./${{ matrix.directory }}/${{ matrix.slug }}
+          ASSETS_DIR: ./${{ matrix.directory }}/${{ matrix.slug }}/.wordpress-org
```

Comment:
The `ASSETS_DIR` variable is what is used by [the 10up deploy action](https://github.com/10up/action-wordpress-plugin-deploy?tab=readme-ov-file#optional-environment-variables) to tell the action where to find the asset directory for whatever is being deployed. Since we're declaring a different build directory, we also need to declare a different assets directory, otherwise the assets from Performance Lab would get deployed to each plugin.

Reply:
Thanks @joemcgill for answering.

Reply:
@joemcgill The workflow works similarly to how it did for the `2.9.0` release https://github.com/WordPress/performance/actions/runs/7978814725/job/21784762898. Could you please point out what's not working?

Reply:
@westonruter Regarding your question, yes due to the `.gitattributes` file the `.wordpress-org` directories should not be part of the ZIP builds. At least this is already the case for modules built into plugins (via `/modules/**/.wordpress-org export-ignore`).

@mukeshpanchal27 We need to make sure the `/plugins/**/.wordpress-org` directories are ignored for the plugin ZIP / deployment as well.

Reply:
Understood, I've added it to the exclude list.